rdrf/report/schema.py

```diff
@@ -231,7 +232,7 @@ def cde_resolver(cdes, _info, cde_model):
 
 
 def get_form_fields(form_key, section_models, section_cdes):
-    fields = {}
+    fields = {'last_updated': graphene.DateTime(description="Timestamp this form was last updated")}
```

Comment:
Causes problems if there's a section called `lastUpdated`

We might need to restructure the gql in order to avoid this like so:

```
clinicalData {
  cfgs {
    CFG_NAME {
      forms {
        FORM_NAME {
          lastUpdated,
          sections {
            SECTION_NAME {
              cdes {
                CDE_NAME
              }
            }
          }
        }
      }
    }
  }
}    

```